api_gateway/src/livekit/livekit.py

```
@@ -91,10 +256,70 @@ class VoiceInterruptionRequest(BaseModel):
-        # Generate session ID
-        session_id = f"voice_session_{int(time.time())}_{request.participant_name}"
+        logger.info(f"=== Creating NEW VOICE SESSION ===")
+        logger.info(f"Force fresh start: {request.force_fresh_start}")
+        logger.info(f"Conversation ID: {request.conversation_id}")
+        logger.info(f"User Profile ID: {request.user_profile_id}")
+        logger.info(f"Individual ID: {request.individual_id}")
+        logger.info(f"Session reset timestamp: {request.session_reset_timestamp}")
+        
+        # Generate unique session ID with extra randomness to prevent collisions
+        import random
+        session_id = f"voice_session_{int(time.time())}_{random.randint(1000, 9999)}_{request.participant_name}"
```

Ruff (0.12.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)